test/onnx/test_operators.py

```diff
@@ -265,6 +265,15 @@ def test_conv_onnx_irv4(self):
         x = torch.ones(20, 16, 50, 40, requires_grad=True)
         self.assertONNX(nn.Conv2d(16, 13, 3, bias=False), x)
 
+    def test_conv_onnx_irv4_opset8(self):
+        # This test point checks that for opset 8 (or lower), even if
+        # keep_initializers_as_inputs is set to False, it is ignored,
+        # and initializers are listed as ONNX graph input, in accordance
+        # with ONNX IR v3 semantics (which apply to opset version <= 8).
+        x = torch.ones(20, 16, 50, 40, requires_grad=True)
+        self.assertONNX(nn.Conv2d(16, 13, 3, bias=False), x, opset_version=8,
```

Comment:
can we use a smaller kernel and zero them, otherwise, we may problem due to the randomness in the initializers.

Reply:
Yes, that makes sense. I have updated the PR to use a smaller kernel and all ones weights. Thanks.